packages/ens-utils/package.json

```diff
@@ -35,18 +35,17 @@
     "access": "public"
   },
   "dependencies": {
-    "@adraffy/ens-normalize": "1.10.1",
+    "@adraffy/ens-normalize": "1.11.0",
     "@ethersproject/hash": "5.7.0",
     "date-fns": "3.3.1",
     "decimal.js": "10.4.3",
```

Comment:
I purposefully don't want to upgrade `date-fns` or `decimal.js` in this PR. Long story but it's not for this PR.